docs/_documentations/codewind-eclipse-quick-guide.md

```diff
@@ -0,0 +1,167 @@
+2. Right-click your project and select `Open Application`. 
+
+Your application is now opened in the browser, showing the welcome to your Appsody microservice page. 
+
+### Adding a REST service to your application
+1. Go to your project's workspace under the Project Explorer tab. 
+2. Navigate to `Java Resources->src/main/java->dev.appsody.starter`. 
+3. Right-click `dev.appsody.starter` and select `New->Class`.
+4. Create a Class file, name it `Calculator.java`, and select `Finish`. This file is your JAX-RS resource. 
+5. Populate the file with the following code then **save** the file: 
+
+```
+package dev.appsody.starter;
+
+import javax.ws.rs.core.Application;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import javax.ws.rs.PathParam;
+
+@Path("/calculator")
+public class Calculator extends Application {
+
+    @GET
+    @Path("/aboutme")
+    @Produces(MediaType.TEXT_PLAIN)
+    public String aboutme(){
+        return "You can add (+), subtract (-), and multiply (*) with this simple calculator.";
+    }
+
+    @GET
+    @Path("/{op}/{a}/{b}")
+    @Produces(MediaType.TEXT_PLAIN)
+    public Response calculate(@PathParam("op") String op, @PathParam("a") String a, @PathParam("b") String b)
+    {
+        int numA = Integer.parseInt(a);
+        int numB = Integer.parseInt(b);
+
+      switch(op)
+      {
+          case "+":
+              return Response.ok(a + "+" + b + "=" + (Integer.toString((numA + numB)))).build();
+
+          case "-":
+              return Response.ok(a + "-" + b + "=" + (Integer.toString((numA - numB)))).build();
+
+          case "*":
+              return Response.ok(a + "*" + b + "=" + (Integer.toString((numA * numB)))).build();
+
+          default:
+              return Response.ok("Invalid operation. Please Try again").build();
+      }
+    }
+}
+```
+Any changes you make to your code is automatically built and re-deployed by Codewind and viewed in your browser.
+
+### Working with the microservice
+You now can work with your calculator. 
+
+* Use the port number you saw when you first opened the application.
+* Make sure to remove the `< >` symbol in the URL. 
+* `http://127.0.0.1:<port>/starter/calculator/aboutme` 
+* You should see the following response:
+
+```
+You can add (+), subtract (-), and multiply (*) with this simple calculator.
+```
+
+You could also try a few of the sample calculator functions:
+
+* `http://127.0.0.1:<port>/starter/calculator/{op}/{a}/{b}`, where you can input one of the available operations `(+, _, *)`, and an integer a, and an integer b.
+* So for `http://127.0.0.1:<port>/starter/calculator/+/10/3` you should see: `10+3=13`. 
+
+## What you have learned 
+In this quick guide, you have learned to:
+1. Installed Codewind on Eclipse
+2. Developed your own microservice using Codewind
+
+## Next Steps 
+See other quick guides to learn how to develop with Codewind.
```

Comment:
Would be nice to link back to the Guides page here.

Reply:
Agreed! But I don't know what link that page shall be yet...